src/electron/ipc/preload.ts

```
@@ -1,39 +1,23 @@
-import { contextBridge, ipcRenderer } from 'electron';
+import { contextBridge, ipcRenderer } from "electron";
 
-contextBridge.exposeInMainWorld('electronAPI', {
-  sendMessage: (message: string) =>
-    ipcRenderer.invoke('gemini:send-message', message),
+contextBridge.exposeInMainWorld("electronAPI", {
+  streamMessageWithHistory: (messages: any[]) => ipcRenderer.invoke("stream-message-with-history", messages),
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)